SPARQLWrapper/sparql_dataframe.py

```diff
@@ -1,50 +1,69 @@
 """
 Query a SPARQL endpoint and return results as a Pandas dataframe.
 """
-from SPARQLWrapper import SPARQLWrapper2, SPARQLWrapper, CSV, SELECT
+import io
+from typing import Any, Dict, List, Union
+
+import rdflib.term
+
+from SPARQLWrapper.SmartWrapper import Bindings, SPARQLWrapper2, Value
+from SPARQLWrapper.Wrapper import CSV, SELECT, SPARQLWrapper
+
 
 class QueryException(Exception):
     pass
 
-def get_sparql_dataframe_orig(endpoint, query):
-    """ copy paste from: https://github.com/lawlesst/sparql-dataframe """
+
+def get_sparql_dataframe_orig(endpoint: str, query: Union[str, bytes]) -> Any: # -> pd.DataFrame:
+    """copy paste from: https://github.com/lawlesst/sparql-dataframe"""
     # pandas inside to avoid requiring it
     import pandas as pd
```

Comment:
For me this line is causing a mypy error:

```
20220219T210014 iwana@host:~/sw/d/github.com/iafork/sparqlwrapper
$ .venv/bin/python3 -m mypy --show-error-context --show-error-codes
SPARQLWrapper/sparql_dataframe.py:20: error: Skipping analyzing "pandas": module is installed, but missing library stubs or py.typed marker  [import]
        import pandas as pd
    ^
SPARQLWrapper/sparql_dataframe.py:20: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 7 source files)
```


```suggestion
    import pandas as pd  # type: ignore[import]
```

I'm not entirely sure why this error does not occur in other places though, but maybe it is specific to my setup somehow.

Reply:
Maybe check what happens once mypy is running in the CI pipeline